net/netfilter/nf_tables_api.c

```diff
@@ -7156,8 +7178,11 @@ static void nf_tables_flowtable_notify(struct nft_ctx *ctx,
 	if (skb == NULL)
 		goto err;
 
+	if (ctx->flags & (NLM_F_CREATE | NLM_F_EXCL))
+		flags |= ctx->flags & (NLM_F_CREATE | NLM_F_EXCL);
+
 	err = nf_tables_fill_flowtable_info(skb, ctx->net, ctx->portid,
-					    ctx->seq, event, 0,
+					    ctx->seq, event, flags,
```

Comment:
As with above centos8 made the changes to set the local parameter `flags` but never seems to use it.
https://github.com/ctrliq/kernel-src-tree/blob/centos_kernel-4.18.0-534.el8/net/netfilter/nf_tables_api.c#L7256

I think we should be i can't understand why they're doing this.

Reply:
> As with above centos8 made the changes to set the local parameter `flags` but never seems to use it. https://github.com/ctrliq/kernel-src-tree/blob/centos_kernel-4.18.0-534.el8/net/netfilter/nf_tables_api.c#L7256
> 
> I think we should be i can't understand why they're doing this.

Removed from the updated PR - Using the flags causes problems so I guess that's why Red Hat didn't use them. It's bizarre code from Red Hat.